### PR TITLE
separate clearing for heat map and virginmap

### PR DESCRIPTION
**Pull request by cp-codes2**
noticed a suggestion in dev and bugs for this. sounds like a good idea to make them clear separately.

### CODE REVIEW

File: resources/public/pxls.js [hunk 1]
```diff
@@ -2132,7 +2132,7 @@ window.App = (function() {
               return;
             }
 
-            if (evt.key === 'o' || evt.key === 'O' || evt.which === 79) { // O key
+            if (evt.key === 'u' || evt.key === 'U' || evt.which === 79) { // U key
```
Comment (netux) [inline resources/public/pxls.js:2135]: Key code for 'U' is 85, for 'u' it is 117.

Comment (netux) [reply resources/public/pxls.js:2135]: While you're at it, can you add a check for when `evt.which === 111` on the one that checks for O (heatmap clear)?

**Comment by netux** [comment]
You missed updating the Keybinds tab under Settings
image

**Comment by cp-codes2** [comment]
👍 always miss stuff like that :notlikethis: will do

**Comment by cp-codes2** [comment]
ok everything thus far has been fixed though `evt.which === 111` has not yet been added

**Pull request merged**